### ISSUE TITLE
Fedora 25: Error: nothing provides bzip2-devel.x86_64 needed by eosio

### ISSUE THREAD

**Issue by liviu-** [EOSIO/eos#6066]
Following the README.md update, I tried the commands recommended [here](https://github.com/EOSIO/eos#rpm-package-install) as shown below:

```
$ cat /etc/fedora-release                                                                       
Fedora release 25 (Twenty Five)                                                                   
$ wget https://github.com/eosio/eos/releases/download/v1.4.1/eosio-1.4.1.x86_64-0.x86_64.rpm    
$ sudo dnf install --allowerasing ./eosio-1.4.1.x86_64-0.x86_64.rpm                             
  Last metadata expiration check: 0:48:11 ago on Fri Oct 19 10:52:08 2018.                          
  Error: nothing provides bzip2-devel.x86_64 needed by eosio-1.4.1.x86_64-0.x86_64                  
```

**Comment by conr2d** [comment]
Fedora 25 is in end-of-life status. You had better upgrade your system.

**Comment by liviu-** [comment]
I should, yeah, but it's on the list of  EOSIO supported OS, no?

**Comment by liviu-** [comment]
Thanks for the update. I tried the new version recommended in the `README.md` file as shown below:

    $ sudo dnf install --allowerasing ./eosio-1.4.4-1.fc27.x86_64.rpm 
    Last metadata expiration check: 1:44:06 ago on Wed Nov 28 12:21:08 2018.
    Error: nothing provides libcrypto.so.1.1()(64bit) needed by eosio-1.4.4-1.x86_64

**Comment by jgiszczak** [comment]
Fedora 25 is only supported by building from source.  The earliest binary release requires Fedora 27.  That binary release is now deprecated and will no longer be available with the v1.6.0 release.  At that time binary releases will only be available for Fedora 28 and later.  Building from source may still work but that too will become unsupported.
